src/sensai/data_transformation.py

```diff
@@ -181,6 +272,9 @@ def __init__(self, columns: Optional[Union[str, Sequence[str]]], categories: Uni
         :param ignoreUnknown: if True and an unknown category is encountered during transform, the resulting one-hot
             encoded columns for this feature will be all zeros. if False, an unknown category will raise an error.
         """
+        super().__init__()
+        self._paramInfo["columns"] = columns
+        self.inferCategories = categories is None
```

Comment:
This should be added to _paramInfo and "categories is not None" should be used below

Reply:
Done
Aaaand merging this :)